Common/Extensions/ExceptionExtensions.cs

```diff
@@ -49,20 +49,6 @@ public static class ExceptionExtensions
         /// </summary>
         public const System.Exception NilException = null;
 
-        /// <summary>
-        /// Check if we are in a exception unwind scenario or not.
-        /// </summary>
-        public static bool InException
```

Comment:
I think Obsolete here also would help, this was never really used but it used to serve the purpose of `how to check if code is in a throw state`